### PR TITLE
Clarify passphrase for openssl_encrypt and openssl_decrypt

### PR DESCRIPTION
**Pull request by bukka**
Extend info for passphrase behaviour and note that it doesn't use KDF.

### CODE REVIEW

File: reference/openssl/functions/openssl-decrypt.xml [hunk 1]
```diff
@@ -68,7 +77,9 @@
      <term><parameter>iv</parameter></term>
      <listitem>
       <para>
-       A non-NULL Initialization Vector.
+       A non-NULL Initialization Vector. If the IV is shorter than expected, it is padded with
```
Comment (cmb69) [inline reference/openssl/functions/openssl-decrypt.xml:80]: Maybe:
```suggestion
       A non-&null; Initialization Vector. If the IV is shorter than expected, it is padded with
```

File: reference/openssl/functions/openssl-encrypt.xml [hunk 2]
```diff
@@ -70,8 +70,17 @@
...
+       <literal>NUL</literal> characters; if the passphrase is longer than expected, it is
+       silently truncated.
       </para>
+      <caution>
+       <simpara>
+        There is no key derivation function used for <parameter>passphrase</parameter> as its name
+        might suggest. The only operation used is padding with <literal>NUL</literal> characters
+        or truncation if the length is different than expected.
+       </simpara>
+      </caution>
```
Comment (cmb69) [inline reference/openssl/functions/openssl-encrypt.xml:83]: This looks wrong, since this is about the `$iv` parameter; `$passphrase` is documented above.

**Review by cmb69 — COMMENTED** [review]
Thank you!

File: reference/openssl/functions/openssl-encrypt.xml (hunk 2 above)
Comment (bukka) [reply reference/openssl/functions/openssl-encrypt.xml]: ah yeah, it was late... :)

**Pull request merged**